### PR TITLE
Add sigs.yaml and script for generation

### PR DESCRIPTION
**Pull request by dhiller**
In order to create [more detailed devstats](cncf/devstats#272) for KubeVirt I've copied the generation script from kubernetes/community  and created a basic sigs.yaml, so we can create the SQL required for cncf/devstats.

Base for the project urls was a github API query for orgs kubevirt, k8snetworkplumbingwg, nmstate and virtblocks. Feedback to take in more is required.

Output is here: https://gist.github.com/dhiller/88ba550b04c8006418b8268c5bb73b8c

TODO:
* much more fine-tuning regarding the sigs and the subprojects
* add generation of (postprocess_repo_groups.sql)[https://github.com/cncf/devstats/blob/master/util_sql/postprocess_repo_groups.sql]
* prow job executing the script, so we can have a look at how the output is

/cc @fabiand @fgimenez @rmohr @davidvossel

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    - https://raw.githubusercontent.com/kubevirt/qe-tools/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/vmctl/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/cloud-image-builder/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/common-templates/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/ovs-cni/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/machine-remediation/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/cluster-api-provider-external/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/kubevirt-operator/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/cdi-operator/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/cdi-api-reference/master/OWNERS
```
Comment (dhiller) [inline sigs.yaml:35]: Should move to `sig-storage`

File: sigs.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    - https://raw.githubusercontent.com/kubevirt/kubevirt-ssp-operator/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/node-labeller/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/kubevirt-template-validator/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/kvm-info-nfd-plugin/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/cluster-network-addons-operator/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/node-maintenance-operator/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/bridge-marker/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/kubectl-virt-plugin/master/OWNERS
+    - https://raw.githubusercontent.com/kubevirt/krew-index/master/OWNERS
```
Comment (dhiller) [inline sigs.yaml:47]: Should move to `sig-virtctl`

Comment (dhiller) [inline sigs.yaml:47]: Should move to `sig-virtctl`

File: sigs.yaml [hunk 3]
```diff
@@ -0,0 +1,166 @@
+# Copied from github.com/kubernetes/community - used for devstats sql generation
+sigs:
+- dir: sig-compute
+  name: compute
+  subprojects:
```
Comment (fgimenez) [inline sigs.yaml:5]: Would it make sense to have a subproject for each separate repo? If there are different OWNER files in each repo, then include that in the `owners` section, wdyt? 

Anyway, AIUI the end result https://gist.github.com/dhiller/88ba550b04c8006418b8268c5bb73b8c would be the same, is that correct?

**Review by fgimenez — APPROVED** [review]
/lgtm

Great work, small question about the structure of the sigs.yaml file.

File: sigs.yaml (hunk 3 above)
Comment (dhiller) [reply sigs.yaml:5]: Yeah, I think that could make sense. This was just a first shot, so I think there's definitely room for improval here (aka automation of scanning for all OWNERS files per repo and then generating the subprojects from that)

Comment (dhiller) [reply sigs.yaml:5]: I was just pinning this down to get a feel of what you folks think about the idea, I think that the categories aren't quite right ATM

**Comment by dhiller** [comment]
Manually merging this PR as the created PR against cncf/devstats already got merged.

**Pull request merged**